app/Http/Controllers/Admin/HomeworkController.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace App\Http\Controllers\Admin;
+
+use App\Http\Controllers\Controller;
+use App\Entry;
+
+class HomeworkController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $homeworks = Entry::where('activity_id', 1)->paginate(15);
```

Comment:
waarom is dit ID hardcoded? wat gebeurt er als de huiswerkbegeleiding activiteit op wordt verwijderd en opnieuw wordt aangemaakt?